src/hangman/Main.java

```diff
@@ -1,10 +1,60 @@
 package hangman;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
 
 	public static void main(String[] args) {
-		System.out.println("Hello world");
 
-	}
+		Dictionary dictionary = new Dictionary();
+		Hangman hangman = new Hangman();
+
+		System.out.println("Do you want to play with the computer (type 1) or on your own (type 2)?");
+
+		// scan user reply
+		Scanner scannerMode = new Scanner(System.in);
+		int playMode = scannerMode.nextInt();
+
+		while (playMode != 1 && playMode != 2) {
+			System.out.println("Please, choose one of the options");
+			playMode = scannerMode.nextInt();
+		}
+
+		if (playMode == 1 || playMode == 2) {
+
+			// Get a random words from the list
+			int IndexForWords = (int) (Math.random() * dictionary.words.length);
+			String guessWord = dictionary.words[IndexForWords];
```

Comment:
This could be a method on an instance of Dictionary

Reply:
Created public static String getAWord() method in Dictinary